nav2_behavior_tree/include/nav2_behavior_tree/bt_service_node.hpp

```diff
@@ -45,8 +45,9 @@ class BtServiceNode : public BT::CoroActionNode
   // Any BT node that accepts parameters must provide a requiredNodeParameters method
   static const BT::NodeParameters & requiredNodeParameters()
   {
-    static BT::NodeParameters params = {{"service_name",
-      "please_set_service_name_in_BT_Node"}};
+    static BT::NodeParameters params = {
+      {"service_name", "please_set_service_name_in_BT_Node"},
+      {"service_timeout", "10"}};
```

Comment:
It seems like this is the same as `node_loop_timeout_` (effectively since all that's happening in this node is calling a service). Why not just use the `node_loop_timeout_` variable instead of introducing a new one?

Reply:
Actually, I did hesitate about this variable, and thought natural to write as ```<SomeService service_timeout="100"/> ```. Btw,  using the node_loop_timeout_ variable may get confused with the blackboard variable node_loop_timeout. But this is not critical, I'll do that since you think node_loop_timeout_ is better.

Reply:
@crdelsey @mkhansen-intel  what do you think?

My understanding from reading the code is that BT navigator sets the default node loop timeout to 10ms to the blackboard. Then the service/action node base classes read from the blackboard this 10ms. That indicates to me that @mkhansen-intel's suggestion in the ticket to override that variable in the BT tag wouldn't work if I understand the difference between the blackboard an tag parameters correctly. 

So if node loop timeout is set by BT navigator, and then stored in the base classes, the only use of that variable that I see is actually in the timeouts to the service/action servers, not having anything to do with the timeout for the `node` itself. So we can just keep that variable.

If I'm missing another use of that variable `node_loop_timeout` to timeout actual behavior tree nodes, correct me. Or If I'm missing that the blackboard and the registered tag parameters are somehow connected to overload, also correct me.